Tasks/Common/ios-signing-common/ios-signing-common.ts

```diff
@@ -317,26 +321,38 @@ export async function getProvisioningProfileType(provProfilePath: string) {
             throw tl.loc('ProvProfileDetailsNotFound', provProfilePath);
         }
 
-        //get ProvisionsAllDevices - this will exist for enterprise profiles
+        // get ProvisionsAllDevices - this will exist for developer id/enterprise profiles
         let provisionsAllDevices: string = await printFromPlist('ProvisionsAllDevices', tmpPlist);
         tl.debug('provisionsAllDevices = ' + provisionsAllDevices);
         if (provisionsAllDevices && provisionsAllDevices.trim().toLowerCase() === 'true') {
-            //ProvisionsAllDevices = true in enterprise profiles
-            provProfileType = 'enterprise';
+            // Bypass the app store, distribute a signed app directly to end users
+            if (macOSProfile) {
+                provProfileType = 'developer-id';
+            }
```

Comment:
OK, so I know this is a nit, but could you please make the `else {` blocks be consistent?  Most in this file use the `} else {` style.  (I'm okay with either; I'd prefer not to have both.)